src/functions/getJournal/framework/__tests__/handler.spec.ts

```diff
@@ -72,4 +77,34 @@ describe('getJournal handler', () => {
       expect(createResponse.default).toHaveBeenCalledWith('No staffNumber provided', 400);
     });
   });
+
+  describe('given there is no employeeId in the authorisation token', () => {
+    it('should indicate a bad request', async () => {
+      dummyApigwEvent.headers = {
+        'Content-Type': 'application/json',
+        Authorization: tokens.employeeId_null,
+      };
+      createResponseSpy.and.returnValue({ statusCode: 400 });
```

Comment:
Shouldn't we respond 403 Forbidden in cases where we can't authorise the request?

Reply:
Yeah, good shout. Chris summarised the conventions [here](https://wiki.i-env.net/display/MES/REST+API+Conventions) so really it should be a 401 for invalid token (null employeeId) and 403 when the employeeId differs.

Reply:
It's an interesting one, depends on whether we want to consider a missing employeeId as an invalid token. I'd be inclined to say that we should, therefore 401 for null/missing as you said. 👍

Reply:
Actually probably both 403. The 401 is inappropriate because the token _is_ valid, it's just that the employeeId is missing.